### PR TITLE
Resolved MisconfigurationException in glow_tts

### PR DESCRIPTION
**Pull request by Qiliang**
end of epoch will throw execption:

pytorch_lightning.utilities.exceptions.MisconfigurationException: ModelCheckpoint(monitor='val_loss') not found in the returned metrics: ['loss', 'l_mle', 'l_length', 'logdet']. HINT: Did you call self.log('val_loss', value) in the LightningModule?

### CODE REVIEW

**Comment by blisc** [comment]
Can you sign your commits? See the output of DCO bot [here](https://github.com/NVIDIA/NeMo/pull/2603/checks?check_run_id=3236746010).
